lib/formatters/codeframe.js

```diff
@@ -47,7 +47,7 @@ function formatFilePath(filePath, line, column) {
  */
 function formatMessage(message, parentResult) {
     const type = (message.fatal || message.severity === 2) ? chalk.red("error") : chalk.yellow("warning");
-    const msg = `${chalk.bold(message.message.replace(/\.$/, ""))}`;
+    const msg = message.message.search(/ \.$/) >= 0 ? `${chalk.bold(message.message)}` : `${chalk.bold(message.message.replace(/\.$/, ""))}`;
```

Comment:
Nitpick: I think it would be simpler to do something like

```js
const msg = chalk.bold(message.message.replace(/[^ ]\.$/, "");
```

rather than using a ternary expression and another regex.